csaf_2.1/prose/edit/src/tests-01-mndtr-13-purl.md

```diff
@@ -1,6 +1,6 @@
-### PURL
+### purl
```

Comment:
Nit: I think, in titles we should add Title Text before any artificial term, as this looks wrong to me (a single word title in non-title case).

Reply:
I was also wondering but wanted to flag it so that we don't accidentally forget it - let's revert it:
```suggestion
### PURL

Reply:
If you are okay with that, feel free to commit the change.